pipeline/vars/v3.groovy

```diff
@@ -36,7 +17,7 @@ def writeToReleaseFile(
 }
 
 def sendEmail(
-    def run_type,
+    def run_type = null,
```

Comment:
Initialisation should not be required since we usually need to pass this. even from cvp and test executor we are passing a value.

Reply:
@manasagowri 
In cpv.groovy we are not using sendEmail method.
In test executor and released.groovy we are not passing the run_type, and as per my understanding there is no need to pass run_type.

Reply:
Okay then you can pass an empty string maybe. Because I feel passing null could cause an issue in sendemail method.

Reply:
ok. Passing an empty String.